components/gc-follow-us/include.html

```diff
@@ -1,9 +1,17 @@
 <section class="provisional gc-followus">
 	<h2>
-		{%- if i18nText-lang == "fr" -%}
-			Dans les médias sociaux
+		{%- if page.followUsHeading -%}
```

Comment:
You were right, let's base the text on the layout. For now and maybe for ever, the only use cases we have is "On social media" when it is vertical and "Follow us" when it is horizontal. Let's go with the simpler version and remove `followUsHeading` then.